framework/web/User.php

```diff
@@ -568,6 +568,10 @@ public function switchIdentity($identity, $duration = 0)
             return;
         }
 
+        /* Ensure any existing identity cookies are removed. */  
+        if ($this->enableAutoLogin)  
+            Yii::$app->getResponse()->getCookies()->remove(new Cookie($this->identityCookie));  
```

Comment:
Always use braces for single line statements.

https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#54-control-statements